src/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,99 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #TAGS
+\*------------------------------------*/
+
+/**
+ * The rounded rectangle container that houses information.
+ * 1) Grabs colors from whichever variant is selected.
+ * 2) Rounds the corners of the tag.
+ */
+.tag {
+  width: max-content;
+  max-width: 20rem;
+  padding-top: var(--eds-size-half);
+  padding-bottom: var(--eds-size-half);
+  padding-left: var(--eds-size-1-and-half);
+  padding-right: var(--eds-size-1-and-half);
+
+  display: inline-flex;
+  align-items: center;
+
+  border: var(--tag-secondary-color) solid var(--eds-theme-border-width); /* 1 */
```

Comment:
I'm personally in favor of using internal css variables to reduce repetition. Interestingly, the `--tag-primary-color` is only used in one place (`color`), so we're actually not getting any benefit from that one (might as well just set the property instead of setting the variable value). I wonder if we're setting all 3 for consistency.

Reply:
Agree for both using internal css variables and that setting `--tag-primary-color` for consistency. Can change that since its used once, but I like the consistency